src/components/Organisms/Header/Header.jsx

```diff
@@ -0,0 +1,47 @@
+import css from './Header.module.css';
+
+import { ReactComponent as HeaderLogo } from '../../../images/icons/logo-header-icon.svg';
+import { ReactComponent as BurgerMenu } from '../../../images/icons/burger-icon.svg';
+import { ReactComponent as Switch } from '../../../images/icons/switch-light-icon.svg';
+
+import { Navbar } from 'components/Molecules/Navbar/Navbar.jsx';
+
+import { Link } from 'react-router-dom';
+import { UserLogo } from 'components/Molecules/UserLogo/UserLogo.jsx';
+import { useState } from 'react';
+import { Navigation } from '../Navigation/Navigation';
+
+const Header = ({ openMenu }) => {
+  const [nav, setNav] = useState(false);
+  let width = window.innerWidth;
```

Comment:
Ask @AnnaIstochnikova  for checking media query handler, she implemented it 💡

Reply:
done